static/templates/settings/notification_settings_checkboxes.hbs

```diff
@@ -2,8 +2,9 @@
     <label class="checkbox">
         <input type="checkbox" name="{{setting_name}}" id="{{prefix}}{{setting_name}}"
           {{#if is_disabled}}disabled="disabled"{{/if}}
+          {{#is_muted}}disabled="disabled"{{/is_muted}}
           {{#if is_checked}}checked="checked"{{/if}} data-setting-widget-type="boolean"
-          class="{{setting_name}}{{#unless is_disabled}} prop-element{{/unless}}"/>
+          class="{{setting_name}}{{#unless is_disabled}} prop-element{{/unless}} {{#is_muted}}disabled_checkbox{{/is_muted}}"/>
```

Comment:
We do not require this `disabled_checkbox` class. We can directlty use `input[disabled]` to get disabled checkboxes in the JS code.